linux/roles/onprem/tasks/users.yml

```
@@ -33,3 +33,8 @@
   user:
     name: ubuntu
     state: absent
+- name: delete users who should no longer have access
+  user:
```

suggestion: `ansible-lint` will want this to be the full name:
```suggestion
  ansible.builtin.user:
```